docs/swagger-ui/src/main/scala/sttp/tapir/swagger/SwaggerUI.scala

```diff
@@ -32,11 +32,13 @@ object SwaggerUI {
     */
   def apply[F[_]](yaml: String, options: SwaggerUIOptions = SwaggerUIOptions.default): List[ServerEndpoint[Any, F]] = {
     val prefixInput: EndpointInput[Unit] = options.pathPrefix.map(stringToPath).reduce[EndpointInput[Unit]](_.and(_))
-    val prefixFromRoot = (options.contextPath ++ options.pathPrefix).mkString("/")
+    val prefixFromRoot = if (options.useRelativePath) "." else "/" + (options.contextPath ++ options.pathPrefix).mkString("/")
 
     val baseEndpoint = infallibleEndpoint.get.in(prefixInput)
     val redirectOutput = statusCode(StatusCode.PermanentRedirect).and(header[String](HeaderNames.Location))
 
+    val lastSegmentInput: EndpointInput[Option[String]] = extractFromRequest(request => request.pathSegments.lastOption)
```

Comment:
Not sure if #extractFromRequest can be used : 
From docs: 
> /** Extract a value from a server request. This input is only used by server interpreters, it is ignored by documentation interpreters and
    * the provided value is discarded by client interpreters.
    */
    
 So looks like it will be skipped. @adamw Is it true?

Reply:
Well the result is a `ServerEndpoint` so it will be interpreted using a server interpreter, so it will work just fine :)

Reply:
Yup, I tested it on a real project and it does work.